### ISSUE TITLE
Login module quirks, Firefox wrap

### ISSUE THREAD

**Issue by AndyGaskell** [C-Lodder/lightning#23]
This is a lovely simple template, I think it'll be my go-to as a base for writing new templates now.  Thanks for making it, lovely work :) 

Just some really minor feedback, with a view to helping, rather than to criticise.

This is from https://demo.joomjunk.co.uk/ just now, not sure if that is the current master in git.

So, in Firefox 74.0 (64-bit) on Ubuntu 19.10 at 100% view, the show/hide password button wraps, like this...
![ffox_login_module](https://user-images.githubusercontent.com/91541/77157877-2666ca80-6a9a-11ea-9221-003d14ca9947.png)

So, in Chrome Version 80.0.3987.132 (Official Build) (64-bit) on Ubuntu 19.10 at 100% view, it looks like this...
![chrome_login_module](https://user-images.githubusercontent.com/91541/77157889-2c5cab80-6a9a-11ea-8c88-6bdd920bf68d.png)

Also, not sure, should it have a label or an icon or something.  Maybe it's missing the fa eye icon, looking at the code.  I know that might just be dependent on Joomla FA bits and decisions, but just to mention it while here.

Thanks again, for the awesome template. Hope you're well.

**Comment by C-Lodder** [comment]
Thanks Andy.

I've fixed the wrapping issue here: https://github.com/C-Lodder/lightning/commit/0f475e2823b08f603764eef642f6fd15719f111d

Will release a proper update once I've added support for top navigation submenus

>  Maybe it's missing the fa eye icon

I haven't added any icons yet as I was waiting for feedback on https://github.com/joomla/joomla-cms/pull/28351 which allowed using SVGs instead of icon fonts. I may temporarly load FA until something more concrete is merged in core.
